cli/script/command-executor.ts

```diff
@@ -621,6 +647,26 @@ function printAccessKeys(format: string, keys: AccessKey[]): void {
     }
 }
 
+function printDeployment(command: cli.IDeploymentViewCommand, deploymentKey: DeploymentKey, packageHistory: Package[]): void {
+    packageHistory.reverse(); // Reverse chronological order
```

Comment:
That's fine for when it's in tabular format and I have to iterate through it again anyway, but I still want to show it in reverse order when I stringify it for the JSON view. I could reverse it in one case and not the other, but this is just more tidy. It's running on the client anyway so the tiny perf hit won't really matter too much.